common/src/test/java/org/apache/rocketmq/common/protocol/header/FastCodesHeaderTest.java

```diff
@@ -65,9 +75,11 @@ public void testDecodeSendMessageHeaderV2() throws Exception {
         return extFields;
     }
 
-    private void check(RemotingCommand command, List<Field> fields) throws Exception {
-        SendMessageRequestHeaderV2 o1 = (SendMessageRequestHeaderV2) command.decodeCommandCustomHeader(SendMessageRequestHeaderV2.class);
-        SendMessageRequestHeaderV2 o2 = AbstractSendMessageProcessor.decodeSendMessageHeaderV2(command);
+    private void check(RemotingCommand command, List<Field> fields,
+            Class<? extends CommandCustomHeader> classHeader) throws Exception {
+        CommandCustomHeader o1 = command.decodeCommandCustomHeader(classHeader);
```

Comment:
If the classHeader is instance of FastCodesHeader, command.decodeCommandCustomHeader(classHeader) also calls fast decode, the result will always be the same